THIRD_PARTY_NOTICES.md

```diff
@@ -300,7 +300,7 @@ THE SOFTWARE.
 This product includes source derived from [generic-pool](https://github.com/coopernurse/node-pool) ([v2.5.4](https://github.com/coopernurse/node-pool/tree/v2.5.4)), distributed under the [MIT License](https://github.com/coopernurse/node-pool/blob/v2.5.4/README.md):
 
 ```
-MIT License Copyright (c) <year> <copyright holders>
+Copyright (c) 2010-2016 James Cooper <james@bitmechanic.com>
```

Comment:
This is very likely to just get overwritten. Do we know why it was generated wrong? Can we update the third party manifest?  Also, the blob above still says MIT, does that matter?

Reply:
Legal approved removing MIT License from the copyright line here.  I'm not sure why it didn't get generated.  I'll take a look at the manifest and try to track that down.

Reply:
It looks like the generic pool module does not have an individual License file and has the license information at the bottom of the readme.  The oss cli tool probably did not pick it up then.

Reply:
Interesting. I would fix the metadata above the license text that is inaccurate... likely using package.json as a source.

Unfortunately, we are likely to accidentally revert to the old state in teh future. Need the ability to set metadata in the manifest and tell it to not be updatable automatically.